src/viam/proto/service/motion/__init__.py

```diff
@@ -17,8 +17,6 @@
     MotionConfiguration,
     MoveOnGlobeRequest,
     MoveOnGlobeResponse,
-    MoveOnMapNewRequest,
-    MoveOnMapNewResponse,
```

Comment:
This is an autogenerated file, so I hope you didn't do this manually. But given that I don't see a corresponding change in the proto files, I think this was a manual update. 

My suggestion would be to let this file continue to be autogenerated, and delete `MoveOnMapNew` from the other files